packages/design/src/FormManager/FormEdit/components/PageSetEdit.tsx

```diff
@@ -4,13 +4,33 @@ import { type PageSetProps } from '@atj/forms';
 
 import { PatternEditComponent } from '../types';
 
-import PageSet from '../../../Form/components/PageSet';
+import ActionBar from '../../../Form/ActionBar';
+import { useRouteParams } from '../../../FormRouter/hooks';
+
+import { PageMenuEdit } from './PageMenuEdit';
 
 const PageSetEdit: PatternEditComponent<PageSetProps> = ({ previewProps }) => {
+  const { routeParams, pathname } = useRouteParams();
   return (
-    <>
-      <PageSet {...previewProps} />
-    </>
+    <div className="grid-row">
+      <nav className="tablet:grid-col-3 tablet:padding-y-3 tablet:padding-right-4">
+        <PageMenuEdit
```

Comment:
Is it correct that the `PageMenuEdit` component is intended to only be used by the `PageSetEdit` component? If so, could we move it into `PageSetEdit.tsx`, or put `PageMenuEdit.tsx` in a `PageSetEdit` directory?

The page edit UI is a little "special" relative to the other pattern edit UIs, since we assume that it must exist, but keeping it in a child will help once we split these components out into separate, user-injectable modules.